templates/org/settings/actions.tmpl

```diff
@@ -1,15 +1,18 @@
 {{template "base/head" .}}
-<div role="main" aria-label="{{.Title}}" class="page-content organization settings webhooks">
+<div role="main" aria-label="{{.Title}}" class="page-content organization settings actions">
 	{{template "org/header" .}}
 	<div class="ui container">
 		<div class="ui grid">
 			{{template "org/settings/navbar" .}}
 			<div class="ui twelve wide column content">
-				{{template "base/alert" .}}
-				{{template "shared/secrets/add_list" .}}
+				{{if eq .PageType "runners"}}
+					{{template "shared/actions/runner_list" .}}
+				{{else if eq .PageType "secrets"}}
+					{{template "base/alert" .}}
```

Comment:
Why "runners" doesn't need `alert`?

Reply:
Because originally `runners` has alert inside its template, and this was a workarund..  Fixed in https://github.com/go-gitea/gitea/pull/24200/commits/7b3514879b7273e528c3d4b332235a3ee75a9d79

Reply:
So, something unusual means something not-well-designed.

Only if fixing fundamental problems, the code base could become easier to maintain.